src/main/java/com/thebois/controllers/game/StructureController.java

```diff
@@ -1,76 +1,38 @@
 package com.thebois.controllers.game;
 
-import com.badlogic.gdx.math.Vector2;
-import com.badlogic.gdx.scenes.scene2d.ui.Widget;
-
-import com.thebois.Pawntastic;
-import com.thebois.controllers.AbstractInputProcessor;
 import com.thebois.controllers.IController;
 import com.thebois.models.world.World;
-import com.thebois.models.world.structures.StructureType;
-import com.thebois.views.IProjector;
 import com.thebois.views.game.IView;
 import com.thebois.views.game.StructureView;
 
 /**
- * Controller for Structures in the world.
+ * Controller responsible for updating the structures in the view.
  */
-public class StructureController extends AbstractInputProcessor implements IController<IView> {
+public class StructureController implements IController<IView> {
 
     private final World world;
     private final StructureView structureView;
-    private final IProjector projector;
-    private final int tileSize = Pawntastic.getTileSize();
-    private Widget gameWidget;
 
     /**
      * Creates a instance of a Structure Controller.
      *
-     * @param world     The world in which the structures exists.
-     * @param projector Projector used for converting screen coordinates to world coordinates.
+     * @param world The world in which the structures exists.
      */
     public StructureController(
-        final World world, final IProjector projector) {
-
+        final World world) {
```

Comment:
style

Now that there are so few parameters, you can remove the line wrapping here.

Reply:
Yeah, I've noticed it myself a few times, it's quite annoying really.